src/components/PageRouter.js

```diff
@@ -53,7 +53,7 @@ class PageRouter extends React.Component {
       updates.closeable = !!updates.branch.route.closeable
 
       updates.currentPath = newPath
-      if (previousPath !== currentPath) {
+      if (currentPath && previousPath !== currentPath) {
```

Comment:
Can you please add braces to improve readability and avoid future dummy bugs?

Reply:
@sjd78 , thanks, I meant these parenthesis :-)
I understand they are not needed, but why not to help a human when parsing that in the future ...